<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistWindow</class>
 <widget class="QDialog" name="PlaylistWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #e8e8e8;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1,0" columnstretch="0,0,0,1">
   <item row="1" column="0">
    <widget class="QGraphicsView" name="playlistCover">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGraphicsView {
	border: 1px solid #bfbfbf; 
	background-color: rgb(255, 255, 255);
}</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="playlistLabel">
       <property name="text">
        <string>Плейлист</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="playlistName">
       <property name="text">
        <string>название</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="playlistListenBtn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FFD700; /* Желтый цвет фона кнопки */
    border: none; /* Убираем границу */
    border-radius: 15px; /* Закругленные углы с радиусом 20 пикселей */
    color: #FFFFFF; /* Белый цвет текста */
    font-size: 16px; /* Размер шрифта 16 пикселей */
	padding: 5px 20px; /* Внутренние отступы 10 пикселей сверху/низу и 20 пикселей слева/справа */
}

QPushButton:hover {
    background-color: #FFD34C; /* Цвет фона при наведении курсора */
}

QPushButton:pressed {
    background-color: #FFC400; /* Цвет фона при нажатии кнопки */
}

QPushButton:disabled {
    background-color: #C0C0C0; /* Цвет фона для отключенной кнопки */
    color: #808080; /* Цвет текста для отключенной кнопки */
}
</string>
       </property>
       <property name="text">
        <string>Слушать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="4">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QSlider" name="trackSlider">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="pageStep">
        <number>2000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="stopTrackBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MainWindowResources.qrc">
           <normaloff>:/new/prefix1/resources/stop.svg</normaloff>:/new/prefix1/resources/stop.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="repeatButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MainWindowResources.qrc">
           <normaloff>:/new/prefix1/resources/no-repeat.svg</normaloff>:/new/prefix1/resources/no-repeat.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="prevTrackBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MainWindowResources.qrc">
           <normaloff>:/new/prefix1/resources/back.svg</normaloff>:/new/prefix1/resources/back.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="playBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MainWindowResources.qrc">
           <normaloff>:/new/prefix1/resources/play.svg</normaloff>:/new/prefix1/resources/play.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextTrackBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MainWindowResources.qrc">
           <normaloff>:/new/prefix1/resources/forward.svg</normaloff>:/new/prefix1/resources/forward.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mixButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MainWindowResources.qrc">
           <normaloff>:/new/prefix1/resources/mix.svg</normaloff>:/new/prefix1/resources/mix.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="muteBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MainWindowResources.qrc">
           <normaloff>:/new/prefix1/resources/mid-volume.svg</normaloff>:/new/prefix1/resources/mid-volume.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="volumeSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTableWidget" name="playlistTracks">
     <property name="styleSheet">
      <string notr="true">QTableWidget {
    background-color:  #e8e8e8; /* Цвет фона */
    border: none; /* Убрать границу */
    color: #333333; /* Цвет текста */
    selection-background-color: #f9fbff; /* Цвет фона выбранной ячейки */
	selection-color: #333333;
    gridline-color: #E5E5E5; /* Цвет линий сетки */
}

QTableWidget::item {
    padding: 8px; /* Внутренний отступ для содержимого ячейки */
}

QTableWidget QHeaderView {
    background-color:  #e8e8e8; /* Цвет фона заголовков столбцов */
    color: #333333; /* Цвет текста заголовков столбцов */
    border: none; /* Убрать границу */
    font-weight: bold; /* Жирный шрифт для заголовков столбцов */
    text-align: center; /* Выравнивание текста по центру */
}

QTableWidget QHeaderView::section {
    background-color:  #e8e8e8;
	border-radius: 0;
}

QTableWidget::item:hover {
    background-color: #f9fbff; /* Цвет фона ячейки при наведении курсора */
}

QTableCornerButton::section {
    background-color:  #e8e8e8; /* Цвет фона квадратика */
}
</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLineEdit" name="playlistSearchLine">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background-color: rgb(255, 255, 255);
    border: 1px solid #bfbfbf; /* Граница */
    border-radius: 10px; /* Закругленные углы */
    color: #333333; /* Цвет текста */
    font-size: 16px; /* Размер шрифта */
    padding: 8px; /* Внутренние отступы */
}

QLineEdit:focus {
    border: 2px solid #1ED760; /* Граница при фокусе */
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Поиск</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="MainWindowResources.qrc"/>
 </resources>
 <connections/>
</ui>
